### ISSUE TITLE
Build upon the default configuration

### ISSUE THREAD

**Issue by PaulWoitaschek** [arturbosch/detekt#1248]
When I supply an empty configuration file, the defaults are being run.

Now I want to suppress the `MagicNumber` rule:

```
style:
  MagicNumber:
    active: false
```

It seems that detekt now runs nothing at all. Is it possible to use detekt so it only disables this one but keeps all the other defaults?
I trust in this library to provide good defaults and only want to suppress some, without copy-pasting a 500-line configuration and updating each time you change your recommendations / add rules.

**Comment by arturbosch** [comment]
Thats actually a valid point and I would expect the same.
I will look into this.

**Comment by passsy** [comment]
I ran into the same problem. Not good documented but it already works. Try this:

```yaml
# Enable all rules by default
failFast: true

# Configure rules and override default values
style:
  MagicNumber:
    active: false
```

**Comment by PaulWoitaschek** [comment]
As far as I understand this does not enable the defaults. It rather just enables everything.

**Comment by passsy** [comment]
Isn't everything enabled the default?

**Comment by PaulWoitaschek** [comment]
Nope
If you don't set a config this is being used:

https://github.com/arturbosch/detekt/blob/master/detekt-cli/src/main/resources/default-detekt-config.yml

And that's whati want to build upon.

**Comment by arturbosch** [comment]
But everything in default detekt config is generated from the KDoc of the rules which is based on the hard coded defaults in the rule itself. So what @passsy writes should be what you want?
